next.config.mjs

```diff
@@ -11,7 +11,11 @@ const withMDX = nextMdx({
 	},
 });
 
-export default withMDX({
-	pageExtensions: ["js", "jsx", "md", "mdx", "ts", "tsx"],
+const nextConfig = {
 	reactStrictMode: true,
-});
+	experimental: {
+		mdxRs: true,
+	},
+};
+
+export default withMDX(nextConfig);
```

Comment:
[Nit] Is there a reason to move from an inline object inside the `export default withMdx(...)` to a separate variable? Not a blocker in my opinion, just curious 😄 _(with a mild preference for the older way)_

Reply:
the older way it is 👴 [cb34e5e](https://github.com/OpenContributionsProject/opencontributionssite/pull/114/commits/cb34e5ea040d4f9390dd6f3ea6067881fe940fa5)